specification/signalr/resource-manager/Microsoft.SignalRService/stable/2018-10-01/signalr.json

```diff
@@ -861,7 +1083,7 @@
           "readOnly": true
         },
         "type": {
-          "description": "The type of the service - e.g. \"Microsoft.SignalRService/SignalR\"",
+          "description": "The type of the resource - e.g. \"Microsoft.SignalRService/SignalR\", \"Microsoft.SignalRService/SignalR/PrivateEndpointConnections\"",
```

Comment:
>Microsoft.SignalRService/SignalR/PrivateEndpointConnections\" [](start = 98, length = 61)

privateEndpointConnections does not belong here as an example on this definition.  This definition describes a tracked resource specific to your service.  PrivateEndpointConnections are proxy